cpp/src/merge/merge.cu

```diff
@@ -80,9 +80,8 @@ __global__ void materialize_merged_bitmask_kernel(
   auto active_threads = __ballot_sync(0xffffffff, destination_row < num_destination_rows);
 
   while (destination_row < num_destination_rows) {
-    index_type const& merged_idx = merged_indices[destination_row];
-    side const src_side          = thrust::get<0>(merged_idx);
-    size_type const src_row      = thrust::get<1>(merged_idx);
+    index_type const& merged_idx   = merged_indices[destination_row];
+    auto const [src_side, src_row] = merged_idx;
```

Comment:
```suggestion
    auto const [src_side, src_row] = merged_indices[destination_row];
```

`merged_idx` doesn't seem to be used anywhere else, why have it at all?

Reply:
Done :+1: